### PR TITLE
Add GF simplified version of coldpool parameterization and coupling to radiation

### PR DESCRIPTION
**Pull request by haiqinli**
Enter a description of this PR. This should include why this PR was created, and what it does.

1). Add a simplified version of GF coldpool parameterization.
2). Add the GF coupling to radiation (Chaboureau and Bechtold).

It is tested on Jet with 13km over CONUS and 3km over HFIP domain.

### CODE REVIEW

**Review by tanyasmirnova — APPROVED** [review]

**Review by mdtoyNOAA — APPROVED** [review]

**Comment by haiqinli** [comment]
@jderrico-noaa Please hold this PR for a while. We like to complete a debug mode test. Thanks.

File: src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F [hunk 1]
```diff
@@ -215,6 +224,24 @@ subroutine cloudiness_from_MPAS(itimestep,configs,mesh,diag_physics,sfc_input,it
...
+
+    if (convection_scheme=="cu_grell_freitas") then
+       call mpas_pool_get_array(diag_physics,'cldfrac_cu',cldfrac_cu)
+       call mpas_pool_get_array(diag_physics,'qi_cu',qi_cu)
+       call mpas_pool_get_array(diag_physics,'qc_cu',qc_cu)
+       do j = jts,jte
+       do k = kts,kte
+       do i = its,ite
+          if (cldfrac_cu(k,i) .gt. 0.001) then
+             cldfrac_p(i,k,j) = cldfrac_p(i,k,j) + cldfrac_cu(k,i)
```
Comment (joeolson42) [inline src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F]: It looks like the cloud fraction can become > 1. Maybe take the max? or limit it to be <= 1.

**Review by joeolson42 — APPROVED** [review]
just make sure the cloud fraction isn't exceeding 1.

File: src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F (hunk 1 above)
Comment (haiqinli) [reply src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F]: @joeolson42 Thanks for your suggestion. Yes, I will limit it <=1 and update the PR accordingly.

**Comment by haiqinli** [comment]
> @jderrico-noaa Please hold this PR for a while. We like to complete a debug mode test. Thanks.

@jderrico-noaa The PR is updated, and is ready to go. Thanks.

**Review by tanyasmirnova — APPROVED** [review]

**Review by clark-evans — APPROVED** [review]

**Review by barlage — APPROVED** [review]

**Pull request merged**